app/helpers/collection_helper.rb

```diff
@@ -3,12 +3,28 @@ def link_to_collection_members(link_text, document, options={})
     link_to(link_text, catalog_index_path(f: { collection: [document[:id]] }))
   end
 
+  def inner_link_to_collection_members(link_text, document, options={})
+    return catalog_index_path(f: { collection: [document[:id]] })
+  end
+
   def collection_members_enumeration(document)
     if document.collection_members.present?
       "#{pluralize(document.collection_members.total, 'item')} online"
     end
   end
 
+  def text_for_inner_members_link(document)
+    if document.collection_members.present?
+      if document.collection_members.total == 1
+        "View this item"
+      elsif document.collection_members.total == 2
+        "View both items"
+      else
+        "View all #{document.collection_members.total} items"
+      end
+    end
+  end
+
```

Comment:
Using pluralise here didn't handle the cases where we need different wording altogether.